### PR TITLE
Created notification when there are distance

### PR DESCRIPTION
**Pull request by elivina997**

### CODE REVIEW

File: app/src/main/java/com/univer/covid19help/MainFragment.kt [hunk 1]
```diff
@@ -212,12 +218,26 @@ class MainFragment : Fragment() {
...
+            updateLocation(LatLng(curLocation.latitude, curLocation.longitude))
+            this.curLocation = curLocation
             val time = (Date().time and 0x0fffff)/1000
-            binding.loc.text = "LAT:${curLatLng.latitude}  LONG:${curLatLng.longitude} ($time)"
+            binding.loc.text = "LAT:${curLocation.latitude}  LONG:${curLocation.longitude} ($time)"
+            binding.buttonSetHome.isEnabled = true
+
+            homeLocation?.run {
+                val dist = this.distanceTo(curLocation)
+                if (dist> DIST_OUT_HOME){
```
Comment (SergeyPjatkin) [inline app/src/main/java/com/univer/covid19help/MainFragment.kt:231]: let keep one style: keep the same spaces for operators.
if (dist > DIST_OUT_HOME)

File: app/src/main/java/com/univer/covid19help/MainFragment.kt [hunk 2]
```diff
@@ -212,12 +218,26 @@ class MainFragment : Fragment() {
...
+                    if (inHome) {
```
Comment (SergeyPjatkin) [inline app/src/main/java/com/univer/covid19help/MainFragment.kt:232]: Run for each source file "Reformat code". AndroidStudio->Menu->Code->Reformat code

File: app/src/main/java/com/univer/covid19help/MainFragment.kt (hunk 1 above)
Comment (elivina997) [reply app/src/main/java/com/univer/covid19help/MainFragment.kt]: Done

File: app/src/main/java/com/univer/covid19help/MainFragment.kt (hunk 2 above)
Comment (elivina997) [reply app/src/main/java/com/univer/covid19help/MainFragment.kt:240]: Done

**Review by SergeyPjatkin — APPROVED** [review]

**Pull request merged**